src/map/items/item_equipment.cpp

```diff
@@ -43,7 +43,6 @@ CItemEquipment::CItemEquipment(uint16 id)
 
 CItemEquipment::~CItemEquipment()
 {
-    // ни в коем случае не освобождать здесь указатели на модификатоты и спецеффекты. они глобальны.
```

Comment:
This function is empty.
Google claims that comment means "in no case free here pointers to modifiers and special effects. they are global."
Which I would guess that means something for someone.

Reply:
slightly more accurate: `in any case, do not release pointers to modifiers and special effects here. they are global.`

Reply:
its just a warning to not destroy those objects here